float32 angle #NEED DEFINE INTERVAL
string emotion
